examples/huggingface/reverb_config.py

```diff
@@ -0,0 +1,115 @@
+# Following https://huggingface.co/docs/transformers/en/custom_models
+import math
+from typing import Dict, List, Optional
+from transformers import PretrainedConfig
+import numpy as np
+
+
+def cmvn(means: List[float], variance: List[float], count: int):
+    """ Calculate cmvn from stats
+
+    Returns:
+        a numpy array of [means, vars]
+    """
+    for i in range(len(means)):
+        means[i] /= count
+        variance[i] = variance[i] / count - means[i] * means[i]
+        if variance[i] < 1.0e-20:
+            variance[i] = 1.0e-20
+        variance[i] = 1.0 / math.sqrt(variance[i])
+    # cmvn = np.array([means, variance])
+    return [means, variance]
+
+
+class ReverbConfig(PretrainedConfig):
```

Comment:
not sure if this is 100% possible, but could we include .json/.yaml (or any other format) with model files and load the whole configuration of this class?